From 13b8cbc7474152dd2eb4750c151e8e730c62f179 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Thu, 17 May 2018 16:49:14 +0800
Subject: [PATCH 3847/5242] MLK-18298-2 ARM: dts: imx7d: change the pcie phy
 in dts

commit  ffd683a792e5248506fbae6b3e9efe5527d076b5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change the pcie phy region in dts accordingly.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    4 ----
 arch/arm/boot/dts/imx7d.dtsi    |    9 ++-------
 2 files changed, 2 insertions(+), 11 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index ebc304e..50a11aa 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -634,10 +634,6 @@
 	status = "okay";
 };
 
-&pcie_phy {
-       status = "okay";
-};
-
 &pcie {
 	pinctrl-names = "default";
 	reset-gpio = <&extended_io 1 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 0096ea2..ac88020 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -237,12 +237,6 @@
 		status = "disabled";
 	};
 
-	pcie_phy: pcie-phy@306d0000 {
-		  compatible = "fsl,imx-pcie-phy";
-		  reg = <0x306d0000 0x10000>;
-		  status = "disabled";
-	};
-
 	epdc: epdc@306f0000 {
 		compatible = "fsl,imx7d-epdc";
 		interrupts = <GIC_SPI 117 IRQ_TYPE_LEVEL_HIGH>;
@@ -348,8 +342,9 @@
 	pcie: pcie@33800000 {
 		compatible = "fsl,imx7d-pcie", "snps,dw-pcie";
 		reg = <0x33800000 0x4000>,
+		      <0x306d0000 0x10000>,
 		      <0x4ff00000 0x80000>;
-		reg-names = "dbi", "config";
+		reg-names = "dbi", "phy", "config";
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
-- 
1.7.9.5

